src/simpla-element-behavior.html

```diff
@@ -10,185 +10,218 @@
 
   window.SimplaBehaviors = window.SimplaBehaviors || {};
 
-  SimplaBehaviors.Element = {
-    properties: {
+  SimplaBehaviors.Element = function(config) {
+    let {
+          type,
+          getCallback,
+          setCallback,
+          dataProperties,
+          updateOn
```

Comment:
It's viable, but personally I like the event hook - I'll probably be using it for repeaters too, as repeater is dealing with complex data. If it's just dealing with plain props then it's easy e.g. text / boolean / number. But if you're doing mutations on an objects props you have to either tie in Polymer's observing of subproperties, or use events. IMO events are good as it can give you fine grained control over when to actually trigger changes.